model_compression_toolkit/core/runner.py

```diff
@@ -282,11 +287,14 @@ def read_model_to_graph(in_model: Any,
         fw_info: Information needed for quantization about the specific framework (e.g.,
                 kernel channels indices, groups of layers by how they should be quantized, etc.)
         fw_impl: FrameworkImplementation object with a specific framework methods implementation.
+        model_leaf_layers (list): List of the module's custom layers, these layers shouldn't be divided into
```

Comment:
For Pytorch. List of the module's custom layers. These layers shouldn't be divided into their submodules. Please note that their quantization will not be optimized.